include/exadg/operators/operator_base.cpp

```diff
@@ -242,6 +242,20 @@ OperatorBase<dim, Number, n_components>::set_constrained_values_to_zero(VectorTy
   }
 }
 
+template<int dim, typename Number, int n_components>
+bool
+OperatorBase<dim, Number, n_components>::check_constrained_values_are_zero(
+  VectorType const & vector) const
+{
+  for(unsigned int i = 0; i < constrained_indices.size(); ++i)
+  {
+    if(vector.local_element(constrained_indices[i]) > std::numeric_limits<Number>::min())
```

Comment:
I think this doesn't work if you have a negative entry.

Reply:
yes, good point